alibi/explainers/anchor_text.py

```diff
@@ -54,7 +56,7 @@ def _load_spacy_lexeme_prob(nlp: 'spacy.language.Language'):
         # https://github.com/explosion/spaCy/discussions/6388#discussioncomment-331096
         if 'lexeme_prob' not in nlp.vocab.lookups.tables:
             from spacy.lookups import load_lookups
-            lookups = load_lookups(nlp.lang, ['lexeme_prob'])
+            lookups = load_lookups(nlp.lang, ['lexeme_prob'])  # type: ignore
```

Comment:
Apparently type of `nlp.lang` is `Optional[str]` and `load_lookups` expects `str`.